### DISCUSSION TITLE
How can I adjust the vaultwarden configuration so that the default prompt box does not block the text on the page? The default prompt box is too large

### DISCUSSION THREAD

**Discussion by cukewu** [dani-garcia/vaultwarden#5033] [Q&A] [answered]
How can I adjust the vaultwarden configuration so that the default prompt box does not block the text on the page? The default prompt box is too large...

![iShot_2024-10-03_22 04 18](https://github.com/user-attachments/assets/929c2b0b-a5cd-4871-ae26-2239030341f6)
![iShot_2024-10-03_22 07 15](https://github.com/user-attachments/assets/807bf9f1-05d3-493c-ba6b-2880ddda70ee)

**Comment by BlackDex** [comment]
This has nothing to do with Vaultwarden. These are clients from Bitwarden which this project does not maintain.

**Reply by cukewu** [reply, marked as answer]
you could go to Settings > Autofill in the browser extension, and change the option got “Show autofill menu on form fields” to “Off” or to “When autofill icon is selected”.
